app/assets/stylesheets/lifehack_style.scss

```diff
@@ -0,0 +1,118 @@
+.form_text {
+  float: left;
+  font-size: 15px;
+  font-weight: bold;
+  margin: 15px;
+}
+
+.lifehack_details {
+  max-width: 300rem;
+  padding: rem-calc(50);
+}
+
+  .panel {
+
+      h5 {
+
+        font-size: 30px;
+        text-align: center;
+      }
+      h2 {
+    border-style: double;
+    color: $primary-color;
+    font-size: 60px;
+    text-align: center;
+    }
+
+    table tr td {
+      background-color: $smoke;
+      padding: 13px 0.875rem 13px 2.875rem;
+      }
+
+    .creator_name {
+      padding: 0 0 0 0;
```

Comment:
Shorthand form for property `padding` should be written more concisely as `0` instead of `0 0 0 0`